src/components/BlogPost/BlogPost.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardActionArea, CardContent, CardMedia, Typography, Chip } from '@material-ui/core';
+
+export interface IBlogPost {
```

Comment:
1. Use type instead of an interface and remove the prefix from the name (name it just `BlogPostProps`). It's not a mistake - we just have such a naming convention in this project.
2. Extend props by `CardProps` from `@material-ui/core`.